vertx-core/src/main/java/io/vertx/core/net/impl/NetServerImpl.java

```diff
@@ -508,7 +508,7 @@ private void bind(
     if (options.isSsl()) {
       bootstrap.childOption(ChannelOption.ALLOCATOR, PartialPooledByteBufAllocator.INSTANCE);
```

Comment:
We shouldn't have VertxByteBufAllocator.UNPOOLED_ALLOCATOR, VertxByteBufAllocator.POOLED_ALLOCATOR and  `PartialPooledByteBufAllocator` and we shouldn't set any ALLOCATOR in our code; Netty will pick directly `ByteBufAllocator.DEFAULT`, if the user doesn't customize its own.

Reply:
yes but for the moment I think it is safe to behave like that, as it has been made impl and not internal anymore, this seems safe to keep like this for a while

Reply:
To me it's fine - but I still don't see the reason to use a partial one - it will hit again https://github.com/eclipse-vertx/vert.x/pull/5316 if we kept on using the partial allocator.
And if we remove the partial one, there's no reason anymore to have the unpooled singleton as well, since it was used just there...

Reply:
yes that will be done in a subsequent commit that cares about SSL